docs/source/tutorial/basic.rst

```diff
@@ -2,21 +2,586 @@
+
+.. accordion:: Install localhost computer and code
+
+    Let's begin by setting up the computer using the ``verdi computer`` subcommand:
+
+    .. code-block:: console
+
+        $ verdi computer setup -L tutor -H localhost -T local -S direct -w `echo $PWD/work` -n
+        $ verdi computer configure local tutor --safe-interval 5 -n
+
+    The first commands sets up the computer with the following options:
+
+    * *label* (``-L``): tutor
+    * *hostname* (``-H``): localhost
+    * *transport* (``-T``): local
+    * *scheduler* (``-S``): direct
+    * *work-dir* (``-w``): The ``work`` subdirectory of the current directory
+
+    The second command *configures* the computer with a minimum interval between connections (``--safe-interval``) of 5 seconds.
+    For both commands, the *non-interactive* option (``-n``) is added to not prompt for extra input.
+
+    Next, let's set up the code we're going to use for the tutorial:
+
+    .. code-block:: console
+
+        $ verdi code setup -L add --on-computer --computer=tutor -P arithmetic.add --remote-abs-path=/bin/bash -n
+
+    This command sets up a code with *label* ``add`` on the *computer* ``tutor``, using the *plugin* ``arithmetic.add``.
+
+.. note::
+    A typical real-world example of a computer is a remote supercomputing facility.
+    Codes can be anything from a Python script to powerful *ab initio* codes such as Quantum Espresso or machine learning tools like Tensorflow.
+
+Let's have a look at the codes that are available to us:
+
+.. code:: console
+
+    $ verdi code list
+
+You can see a single code ``add@tutor``, with PK = 5, in the printed list.
+This code allows us to add two integers together.
+The ``add@tutor`` identifier indicates that the code with label ``add`` is run on the computer with label ``tutor``.
+To see more details about the computer, you can use the following ``verdi`` command:
+
+.. code:: console
+
+    $ verdi computer show tutor
+
+Note that the *Work directory* has been set up as the ``work`` subdirectory of the current directory.
+This is the directory in which the calculations running on the ``tutor`` computer will be executed.
+
+Let's now start up the ``verdi shell`` again and load the ``add@tutor`` code using its label:
+
+.. code-block:: ipython
 
-See `issue #3981 <https://github.com/aiidateam/aiida-core/issues/3981>`_.
+    In [1]: code = load_code(label='add')
 
+Every code has a convenient tool for setting up the required input, called the builder.
+It can be obtained by using the ``get_builder`` method:
+
+.. code-block:: ipython
+
+    In [2]: builder = code.get_builder()
+
+Using the builder, you can easily set up the calculation by directly providing the input arguments.
+Let's use the ``Int`` node that was created by our previous ``calcfunction`` as one of the inputs:
+
+.. code-block:: ipython
+
+    In [3]: builder.x = load_node(pk=4)
+       ...: builder.y = Int(5)
```

Comment:
The prompt doesn't look right to me.
```suggestion
    In [3]: builder.x = load_node(pk=4)
    In [4]: builder.y = Int(5)
```

Reply:
This is what the prompt looks like when you copy-paste a code snippet into the IPython shell. Once the `sphinx-copybutton` extension has been supplied with regexes for the various prompts by @chrisjsewell, I think this is what the users will see if they use the button to copy the code block and paste it into the shell. I'm happy to change it though, if you think this prompt is confusing for the user.

Reply:
No, that's ok. As long as it looks exactly like it would if copy & pasted, everything is fine.